pom.xml

```diff
@@ -1057,6 +1052,12 @@
                 <artifactId>kotlin-stdlib</artifactId>
                 <version>2.0.0</version>
             </dependency>
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-api</artifactId>
+                <version>5.10.2</version>
+                <scope>test</scope>
```

Comment:
I see that the dependency updating tool has fixed this but for future reference don't include scopes in the dependency management section.  The scope should always be explicitly specified with the dependency itself.